dune/gdt/tools/timestepper/adaptive-rungekutta.hh

```diff
@@ -77,28 +77,22 @@ class AdaptiveButcherArrayProvider<RangeFieldType, TimeStepperMethods::bogacki_s
 public:
   static Dune::DynamicMatrix<RangeFieldType> A()
   {
-    return Dune::XT::Common::from_string<Dune::DynamicMatrix<RangeFieldType>>(
-        "[0 0 0 0; 0.5 0 0 0; 0 0.75 0 0; " + Dune::XT::Common::to_string(2.0 / 9.0, 15) + " "
-        + Dune::XT::Common::to_string(1.0 / 3.0, 15) + " " + Dune::XT::Common::to_string(4.0 / 9.0, 15) + " 0]");
+    return {{0., 0., 0., 0.}, {1. / 2., 0., 0., 0.}, {0, 3. / 4., 0., 0.}, {2. / 9., 1. / 3., 4. / 9., 0.}};
   }
 
   static Dune::DynamicVector<RangeFieldType> b_1()
   {
-    return Dune::XT::Common::from_string<Dune::DynamicVector<RangeFieldType>>(
-        "[" + Dune::XT::Common::to_string(2.0 / 9.0, 15) + " " + Dune::XT::Common::to_string(1.0 / 3.0, 15) + " "
-        + Dune::XT::Common::to_string(4.0 / 9.0, 15) + " 0]");
+    return {2. / 9., 1. / 3., 4. / 9., 0.};
   }
 
   static Dune::DynamicVector<RangeFieldType> b_2()
   {
-    return Dune::XT::Common::from_string<Dune::DynamicVector<RangeFieldType>>(
-        "[" + Dune::XT::Common::to_string(7.0 / 24.0, 15) + " " + Dune::XT::Common::to_string(1.0 / 4.0, 15) + " "
-        + Dune::XT::Common::to_string(1.0 / 3.0, 15) + " " + Dune::XT::Common::to_string(1.0 / 8.0, 15) + "]");
+    return {7. / 24., 1. / 4., 1. / 3., 1. / 8.};
   }
 
   static Dune::DynamicVector<RangeFieldType> c()
   {
-    return Dune::XT::Common::from_string<Dune::DynamicVector<RangeFieldType>>("[0.5 0.75 1 0]");
+    return {0, 0.5, 0.75, 1.};
```

Comment:
Did you change the order here on purpose?

Reply:
Yep, the old one was wrong. That did not make any problems as we are not using the ``c`` anywhere (it is only used for explicitly time-dependent operators, which we do not have at the moment).